### ISSUE TITLE
OSX gtop not resizing window after node upgrade

### ISSUE THREAD

**Issue by embano1** [aksakalli/gtop#58]
#### Environment

- OS: OSX Sierra
- Node version: v8.8.1
- gtop version: 0.1.5

#### Description

After node upgrade (from 8.1.2 to 8.8.1) gtop does not automatically adapt to window resizes. Tried in the default OSX terminal app, iTerm, tmux. Same result. The reason I ran into this, was when I installed gtop on another machine with a recent node version. On the machine with the old node version, the app automatically adjusted to window resizes. After upgrading node also on this machine, it´s not working anymore.

Since I am not a node expert, I´m having a hard time figuring out which module might have caused this change.

**Comment by TimothyEarley** [comment]
I had the problem of resizing too. However, I am not sure whether it was also caused by a node upgrade.
While trying to fix the problem I added a line [here](https://github.com/aksakalli/gtop/blob/149c7fda3289366fea0b85d95b2f710df44c13ce/lib/gtop.js#L81), which seems to have fixed the issue:
```javascript
process.on('SIGWINCH', function() {});
```
I won't open a pull request for this since I have no idea how or why this fix worked, but it would be nice if someone more competent would do so.
